### PR TITLE
Artp 1361 save and restore layout

### PR DESCRIPTION
**Pull request by chuck-adaptive**
This branch builds on the work completed for ARTP-1360 andARTP-1333. We need all views to be managed by the Platform API in order for them to be properly saved/restored in a layout **now includes 1360, 1333, 1361**

* `defaultCentered` was overriding my snapshots. So, despite saving windows in different places as part of a snapshot - the main window was still showing up in the center.

### CODE REVIEW

**Review by garyadaptive — COMMENTED** [review]
Looks good, just a small question about styling.

File: src/client/src/rt-platforms/openfin/components/OpenFinSnapshot/styled.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  cursor: pointer;
+  padding: 0 0.7rem;
+  height: 1.6rem;
+  font-size: 0.65rem;
+  font-weight: 350;
+  margin-right: 0.5rem;
+`
+
+export const Root = styled.div`
+  position: relative;
```
[the remark this answers is not in the document]
Comment (chuck-adaptive) [reply src/client/src/rt-platforms/openfin/components/OpenFinSnapshot/styled.tsx:30]: No UX designs for this one

[the remark this answers is not in the document]
Comment (chuck-adaptive) [reply src/client/src/rt-platforms/openfin/components/OpenFinSnapshot/styled.tsx:30]: After today's standup & knowing the lock icon would be gone anyway, I replaced it with some text

**Review by garyadaptive — APPROVED** [review]
Works like a charm.

**Pull request merged**